snuba/datasets/errors_replacer.py

```diff
@@ -553,12 +577,29 @@ def process_exclude_groups(message: Mapping[str, Any]) -> Optional[Replacement]:
     See docstring in `sentry.reprocessing2` for more information.
     """
 
-    group_ids = message["group_ids"]
-    if not group_ids:
+    project_id: int
+    group_ids: Sequence[int]
+
+    @classmethod
+    def parse_message(
+        cls, message: Mapping[str, Any], context: ReplacementContext
+    ) -> Optional["ExcludeGroupsReplacement"]:
```

Comment:
Since we're on Python 3.7+ you don't need to define the type as a string. Just enable for this module by importing `from __future__ import annotations`